src/tool_cb_hdr.c

```diff
@@ -103,45 +104,45 @@ size_t tool_header_cb(char *ptr, size_t size, size_t nmemb, void *userdata)
   if(per->config->etag_save_file && etag_save->stream) {
     /* match only header that start with etag (case insensitive) */
     if(curl_strnequal(str, "etag:", 5)) {
-      char *etag_h = NULL;
+      char *etag_v = NULL;
       char *first = NULL;
       char *last = NULL;
       size_t etag_length = 0;
 
-      etag_h = ptr;
-      /* point to first occurrence of double quote */
-      first = memchr(etag_h, '\"', cb);
+      /*
+       * move etag pointer to header's value
+       * we have to go to the semicolon and trim all the LWS preceding
+       * the field value https://tools.ietf.org/html/rfc2616#section-4.2
+       */
```

Comment:
these comments are unnecessary you can infer what is happening from the code

Reply:
Thanks for your review. 
I agree with all the other ones but imo, as a newcomer, having a comment explaining why we are looping over the header adds context :thinking:?

Reply:
IMO no, it's straightforward. Let's see if anyone else has an opinion on it. also you could just use the loop to do the double quote check here and assign first and last rather than have it in a separate function

Reply:
Ok. I agree that I could have used the loop and do the check here but this way I think it's clearer... just by *reading* the function name you see that having a double quoted etag header is mandatory.
But tell me if we have common practices/context to add new function :+1: